client/auth/base/api.js

```diff
@@ -167,7 +170,24 @@ define([
      *   Email address used to pre-fill into the account form,
      *   but the user is free to change it.
      */
-    signUp: partial(authenticate, Constants.SIGNUP_ACTION)
+    signUp: partial(authenticate, Constants.SIGNUP_ACTION),
+
+    /**
+     * Best choice auth strategy, has no action set
+     *
+     * @method bestChoice
+     * @param {Object} config - configuration
+     *   @param {String} config.state
+     *   CSRF/State token
+     *   @param {String} config.redirectUri
+     *   URI to redirect to when complete
+     *   @param {String} config.scope
+     *   OAuth scope
+     *   @param {String} [config.email]
```

Comment:
if `email` is specified and that matches the previously signed in email, does the user still have the option to modify the email?